deepspeed/launcher/launch.py

```diff
@@ -134,7 +135,11 @@ def sigkill_handler(signum, frame):
             except Exception:
                 pass
         if last_return_code is not None:
-            raise subprocess.CalledProcessError(returncode=last_return_code, cmd=cmd)
+            try:
+                raise subprocess.CalledProcessError(returncode=last_return_code, cmd=cmd)
```

Comment:
There is no need to raise a known exception and catch it in order to print the message. Just print enough message here. @jeffra what do you think?

Reply:
I agree, printing seems fine instead. Perhaps printing to the error logger if it’s non zero might be nice?

Reply:
Sure. use `logger.error` is better here.